base/applications/notepad/dialog.c

```diff
@@ -942,7 +942,35 @@ VOID DIALOG_StatusBarUpdateCaretPos(VOID)
     col = ((ich < 0) ? 0 : (dwStart - ich));
 
     StringCchPrintf(buff, _countof(buff), Globals.szStatusBarLineCol, line + 1, col + 1);
-    SendMessage(Globals.hStatusBar, SB_SETTEXT, SBPART_CURPOS, (LPARAM)buff);
+
+    /* Get all text from the edit control to count words */
+    INT cchTotalText = GetWindowTextLengthW(Globals.hEdit);
```

Comment:
These codes will synchronously block each caret move.
It might be felt as system slowdown.
Allocation is not necessary.